src/garage/torch/algos/vpg.py

```diff
@@ -232,7 +230,7 @@ def _compute_kl_constraint(self, obs):
 
         return kl_constraint.mean()
 
-    def _compute_policy_entropy(self, obs):
+    def compute_policy_entropy(self, obs):
```

Comment:
will calling this method at an arbitrary time cause the gradients to be wrong if self._stop_entropy_gradient is false? if so, you can't make this method public.